### DISCUSSION TITLE
Download hook?

### DISCUSSION THREAD

**Discussion by CabalCrow** [qutebrowser/qutebrowser#8236] [Q&A] [answered]
Is there a download hook that can run a shell command on a file downloaded by qutebrowser? I want to hook it up to clamav.

**Comment by The-Compiler** [comment, marked as answer]
Not yet, see:

- #30 
- #35

You can however use [`inotifywait`](https://linux.die.net/man/1/inotifywait) to do this, or maybe ClamAV has something on its own to automatically run on all new files in a directory.
